### DISCUSSION TITLE
is there any way to access ContentView‘s lifecycle?

### DISCUSSION THREAD

**Discussion by miao17game** [dotnet/maui#6435] [Q&A] [answered]
I need to unsubscribe my rx subscription when my custon control destroyed, but it seems no lifecycle like 'OnDestroyed' on ContentView.

**Comment by pictos** [comment, marked as answer]
You can listen to the handlerChanged event, if it's null you can assume that control is destroyed. Also, I belive there's a Loaded Unloaded event that you can use as well.
